cypress/support/commands.js

```
@@ -40,4 +40,27 @@ Cypress.Commands.add('ignoreUncaughtException', (expectedMessage) => {
     done();
     return false;
   });
-});
\ No newline at end of file
+});
+
+//If the shared header is blocking click events, hide it with this
+Cypress.Commands.add('hideSharedHeader', () => {
+  cy.get('crds-shared-header').as('sharedHeader');
+  cy.get('@sharedHeader').invoke('attr', 'class').then(($classValues) => {
+    const newClass = `${$classValues} hide`;
+    cy.get('@sharedHeader').invoke('attr', 'class', newClass);
+  });
+});
+
+//Here for convenience but use sparingly - we usually want these to be thrown
+//Given list of regex, will ignore if error matches any
+Cypress.Commands.add('ignoreMatchingErrors', (errorList) => {
+  cy.on('uncaught:exception', (err) => {
```

Use `Cypress.on` here instead of `cy.on` - `cy` is scoped only to tests, so it won't ignore exceptions that are hit during the before/after hooks, but `Cypress` is globally scoped so it'll ignore exceptions in hooks too. This difference is probably why we've had so much trouble with handling uncaught exceptions even when they were placed where we thought they should be.